src/cli/cluster/cluster_manager.js

```diff
@@ -85,14 +85,16 @@ module.exports = class ClusterManager {
     const chokidar = require('chokidar');
     const fromRoot = require('../../utils/fromRoot');
 
-    this.watcher = chokidar.watch([
+    const watchPaths = uniq([
       'src/plugins',
```

Comment:
I'm concerned about passing this directly to `resolve()`, because we don't enforce a specific working directory. We should probably resolve these early (with `resolve(__dirname, relativePath)`) and then also resolve all paths in the `.map()` below.